dbms/src/Storages/Page/WriteBatchImpl.h

```diff
@@ -41,6 +42,10 @@ enum class WriteBatchWriteType : UInt8
     // In V2, it is the same as `PUT`; In V3, we treated it as a different type from `PUT`
     // to get its lifetime management correct.
     PUT_EXTERNAL = 4,
+    // Put a remote page which data is stored on remote storage
+    PUT_REMOTE = 5,
+    // Update local cache for remote page
+    UPDATE_REMOTE = 6,
```

Comment:
Suggest renaming this to be `UPDATE_DATA_FROM_REMOTE`

Reply:
Renamed.